app/models/usps_mail_event.rb

```diff
@@ -0,0 +1,6 @@
+class Event < ActiveRecord::Base
```

Comment:
The class in this file needs to match the file name, so it needs to be `UspsMailEvent` otherwise Rails autoloader will get very very mad